packages/blocks/src/database-block/table/types.ts

```diff
@@ -12,7 +12,42 @@ export interface Column<
   type: ColumnType;
   name: string;
   data: Data;
+  statCalcOp: StatCalcOpType;
```

Comment:
@zzj3720 I need to make a migration right ?

Reply:
We'll try to avoid migration now, so this should be an optional value